src/wilcoxon.jl

```diff
@@ -179,6 +182,43 @@ function pvalue(x::ApproximateSignedRankTest; tail=:both)
     end
 end
 
+ci(x::ApproximateSignedRankTest, alpha::Float64=0.05; tail=:both) = calculate_ci(x.vals, alpha; tail=tail)
+
+function calculate_ci(x::AbstractVector, alpha::Float64=0.05; tail=:both)
```

Comment:
hm, now that I think more about this, I'm not actually sure that this method will work for `ApproximateSignedRankTest`, but going to need to test it out more to see